### PR TITLE
Add a roadmap page and prioritize mentioning that revenue is not yet implemented

### LINKED ISSUE #322: Remove revenue section on front page and introduce a roadmap page
I believe it's misleading to detail a non-existent revenue system, with only a small note at the end of the paragraph saying that there's no ETA for its implementation. However, I understand that this is a relatively new project with lots in store for the future. Perhaps we should introduce a "Roadmap" page that sets out plans and ideas for future features?

### PR DESCRIPTION
**Pull request by MarkusG**
This resolves #322.

Adding a roadmap to the site not only makes it more available, but the included link also draws more members into the Discord server.

Also, I'm brand new to Vue.js, so any criticism is, of course, welcome.

### CODE REVIEW

File: layouts/default.vue [hunk 1]
```diff
@@ -35,6 +35,15 @@
...
                 <span>Mods</span>
               </NuxtLink>
+              <NuxtLink
+                to="/roadmap"
+                class="tab"
+                :class="{
+                  'active-path': this.$route.path.startsWith('/mod'),
+                }"
+              >
+                <span>Roadmap</span>
```
Comment (wafflecoffee) [inline layouts/default.vue:45]: This should preferably be in the footer, not the header.

**Review by wafflecoffee — CHANGES_REQUESTED** [review]
It would also be best for this to target the `apiv2-support` branch rather than the `master` branch - apiv2 has the pending redesign and has a lot of pending changes which may end up conflicting with these changes.

**Comment by MarkusG** [comment]
Rebased my changes onto the `apiv2-support` branch, re-styled the roadmap page to match the about page, and also moved the roadmap link from the navbar into the footer.

**Review by wafflecoffee — APPROVED** [review]

**Review by falseresync — APPROVED** [review]

**Comment by Geometrically** [comment]
Sorry for never reviewing this, if you still want this PR, you can retarget it to master. However, the new homepage will have  a roadmap.

**Pull request closed without merging**